Research/ResearchUI/iOS/Views/RSDRoundedButton.swift

```diff
@@ -155,7 +155,7 @@ import UIKit
         }
         
         // Set the title font to the font for a rounded button.
-        titleLabel?.font = designSystem.fontRules.font(for: .heading4)
+        titleLabel?.font = designSystem.fontRules.font(for: buttonType)
```

Comment:
These are currently statically defined b/c as graphical elements, they need to fit within the circle. To allow for customization of the font family, instead they should use the `font(forSize:weight:)` method described above.